desktop/core/src/desktop/js/parse/jison/sql/dasksql/show/show.jison

```diff
@@ -0,0 +1,83 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+DataDefinition
+ : ShowStatement
+ ;
+
+DataDefinition_EDIT
+ : ShowStatement_EDIT
+ ;
+
+ShowStatement
+ : ShowColumnsStatement
+ | ShowSchemasStatement
+ | ShowTablesStatement
+ ;
+
+ShowColumnsStatement
+ : 'SHOW' 'COLUMNS' 'FROM' SchemaQualifiedTableIdentifier
+ ;
+
+ShowSchemasStatement
+ : 'SHOW' 'SCHEMAS'
+ | 'SHOW' 'SCHEMAS' 'LIKE' SingleQuotedValue
+ ;
+
+ShowTablesStatement
+ : 'SHOW' 'TABLES' 'FROM' DatabaseIdentifier
+ ;
+
+ShowStatement_EDIT
+ : 'SHOW' 'CURSOR'
```

Comment:
I don't remember if needed, but if CURSOR needs to be highlighted there is one way https://docs.gethue.com/developer/parsers/#updating-keywords

Reply:
CURSOR is not a keyword, but it is somewhat a "meta-keyword" used by Hue to tell the parser, where the user has put their cursor. This is needed to make meaningful autocompletion proposals. For example, if you enter "SELECT FROM table" and then go back to the SELECT Hue can make better suggestions on the columns (as the table is already clear).
That is not something I invested, that is already a cool feature of Hue :-)

Reply:
My bad, I forgot about it, makes sense!